### DISCUSSION TITLE
GitHub Search (beta) will not search past column 1024 for content

### DISCUSSION THREAD

**Discussion by Liturgist** [community/community#52114] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

BUG? I could be wrong, but here is an experiment that appears to show it.

Files were created with content 'THE_PATTERN' starting at column 1,000 through 1,500. A fill character, 'y', precedes the content and follows the content. A script to create the test files is in the repo.
```
PatTest-1000.txt
PatTest-1001.txt
PatTest-1002.txt
...
PatTest-1499.txt
```
The following search expression was used. Since the 'THE_PATTERN' content is in all files, one would expect 500 files to be found.

    repo:Liturgist/test003 path:PatTest*.txt content:THE_PATTERN

Only 14 files were found by this search.

**Comment by look** [comment, marked as answer]
You are correct, we do not index long lines currently.

**Reply by Liturgist** [reply]
> You are correct, we do not index long lines currently.

Given the seriousness, this must be given as a limitation. Is there a timeframe for correcting this fundamental capability?
https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations

**Reply by look** [reply]
Thanks for the suggestion, the line length limitation has been [added to the list](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations).
